vision/product_search/add_product_to_product_set_test.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package productsearch
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestAddProductToProductSet(t *testing.T) {
+	tc := testutil.SystemTest(t)
+
+	const location = "us-west1"
+	const productSetDisplayName = "fake_product_set_display_name_for_testing"
+	const productSetID = "fake_product_set_id_for_testing"
+	const productDisplayName = "fake_product_display_name_for_testing"
+	const productCategory = "homegoods"
+	const productID = "fake_product_id_for_testing"
+
+	var buf bytes.Buffer
+
+	// Create fake product set and product.
+	if err := createProductSet(&buf, tc.ProjectID, location, productSetID, productSetDisplayName); err != nil {
+		t.Fatal(err)
```

Comment:
Would you mind adding more context to the `t.Fatal` calls using `t.Fatalf`? The goal is to make it easy to find what line failed. So, something like:
```suggestion
		t.Fatalf("createProductSet: %v", err)
```